examples/bpk-component-rating/examples.module.scss

```diff
@@ -20,4 +20,14 @@
 
 .bpk-rating-story {
   width: 10rem;
+
+  &--img {
+    height: 1rem;
+    margin-right: -$bpk-one-pixel-rem * 4;
```

Comment:
Would like to suggest to use standard spacing directly.
![image](https://user-images.githubusercontent.com/86780745/168227432-d5d72439-18a7-49eb-8285-ab98101506a4.png)

Reply:
Nice catch, use bpk-spacing-v2 could be better